app/boards/arm/preonic/preonic_rev3.keymap

```diff
@@ -0,0 +1,81 @@
+// |       |  F7  |  F8  |  F9  |  F10 |  F11 |  F12  |       | LCTL | HOME |  END  |       |
+// | LALT  |      |      |      |      |      |       |  ADJ  | NEXT | VOL- | VOL+  | PLAY  |
+            bindings = <
+    &kp TILD &kp BANG  &kp ATSN  &kp HASH  &kp LABT  &kp PRCT  &kp CRRT  &kp AMPS  &kp RABT  &kp LPRN  &kp RPRN  &kp BKSP
+    &kp TILD &kp BANG  &kp ATSN  &kp HASH  &kp LABT  &kp PRCT  &kp CRRT  &kp AMPS  &kp RABT  &kp LPRN  &kp RPRN  &kp DEL 
+    &kp DEL  &kp F1    &kp F2    &kp F3    &kp F4    &kp F5    &kp F6    &kp MINUS &kp KPLS  &kp LBKT  &kp RBKT  &kp PIPE
+    &trans   &kp F7    &kp F8    &kp F9    &kp F10   &kp F11   &kp F12   &trans    &kp LCTL  &kp HOME  &kp END   &trans  
+    &kp LALT &trans    &trans    &trans    &trans    &trans    &trans    &mo ADJ   &cp M_NEXT &cp M_VOLD &cp M_VOLU &cp M_PLAY
+            >;
+        };
+        raise {
+// -----------------------------------------------------------------------------------------
+// |   `   |  1   |  2   |  3   |  4   |  5   |   6   |   7   |   8  |  9   |   0   | BSPC  |
+// |   `   |  1   |  2   |  3   |  4   |  5   |   6   |   7   |   8  |  9   |   0   | DEL   |
+// |  DEL  |  F1  |  F2  |  F3  |  F4  |  F5  |  F6   |   -   |   =  |  [   |   ]   |  \    |
+// |       |  F7  |  F8  |  F9  |  F10 |  F11 |  F12  |   #   |   \  | PGUP |  PGDN |       |
+// |       |      |      |      |  ADJ | SHFT |       |       | NEXT | VOL- |  VOL+ | PLAY  |
+            bindings = <
+    &kp GRAV &kp NUM_1 &kp NUM_2 &kp NUM_3 &kp NUM_4 &kp NUM_5 &kp NUM_6 &kp NUM_7 &kp NUM_8 &kp NUM_9 &kp NUM_0 &kp BKSP
+    &kp GRAV &kp NUM_1 &kp NUM_2 &kp NUM_3 &kp NUM_4 &kp NUM_5 &kp NUM_6 &kp NUM_7 &kp NUM_8 &kp NUM_9 &kp NUM_0 &kp DEL
+    &kp DEL  &kp F1    &kp F2    &kp F3    &kp F4    &kp F5    &kp F6    &kp MINUS &kp EQL   &kp LBKT  &kp RBKT  &kp BSLH
+    &trans   &kp F7    &kp F8    &kp F9    &kp F10   &kp F11   &kp F12   &kp HASH  &kp BSLH  &kp PGUP  &kp PGDN  &trans  
+    &trans   &trans    &trans    &trans    &mo ADJ   &kp LSFT  &trans    &trans    &cp M_NEXT &cp M_VOLD &cp M_VOLU &cp M_PLAY
+            >;
+        };
+        adjust {
+// -----------------------------------------------------------------------------------------
+// |   F1  |  F2  |  F3  |  F4  |  F5  |  F6  |  F7   |   F8  |  F9  |  F10 |  F11  |  F12  |
+// |       |RESET |      |      |      |      |       |       |      |      |       |  DEL  |
+// |       |      |      |      |      |      |       |       |      |      |       |       |
+// |       |      |      |      |      |      |       |       |      |      |       |       |
+// |       |      |      |      |      |      |       |       |      |      |       |       |
+            bindings = <
+    &kp F1       &kp F2       &kp F3       &kp F4       &kp F5       &kp F6  &kp F7  &kp F8  &kp F9  &kp F10 &kp F11 &kp F12   
+    &bt BT_SEL 0 &bt BT_SEL 1 &bt BT_SEL 2 &bt BT_SEL 3 &bt BT_SEL 4 &trans  &trans  &trans  &trans  &trans  &trans  &kp DEL
```

Comment:
Thanks for adding this... At the same time... There's no way to *get* to this adjust layer (yet, I'm still brainstorming how best to handle this common pattern). Can we put this on `raise` instead?

Reply:
@shurikai Did you see this comment? TIA.